style.css

```diff
@@ -0,0 +1,346 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Josefin Sans', sans-serif;
+  box-sizing: border-box;
+}
+
+.hero {
+  height: 100vh;
+  width: 100%;
+  background-image: url(../img/background.jpg);
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
this image is not displayed. you should change it

Reply:
- [ ] Kindly ask for changes in a more friendly and polite way. 
for this example, you can say: `Please add the correct image path because the image is not displayed`